addons/point_of_sale/models/pos_config.py

```diff
@@ -587,17 +587,10 @@ def _action_to_open_ui(self):
         }
 
     def _get_url_to_cache(self, debug):
-        url_to_cache = []
-        base_urls = [
-            f"/pos/ui?config_id={self.id}&from_backend=True",
-            f"/pos/ui?config_id={self.id}",
+        url_to_cache = [
+            f"/pos/ui/{self.id}?from_backend=True",
+            f"/pos/ui/{self.id}",
         ]
-
-        if not debug:
-            url_to_cache.extend(base_urls)
-        else:
-            url_to_cache.extend([f"{url}&debug={debug}" for url in base_urls])
-
```

Comment:
I think it's better to keep this logic where we append `debug={debug}` when in debug mode.